frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -256,7 +266,7 @@ async function emailConfirmation(activityId: string) {
     bodyType: 'text',
```

Comment:
The `bodyType` is defined here, and can be set to html as we know the email contents we wish to send. The changes to the email api don't feel necessary.

Reply:
yes that works!